GVFS/GVFS.Common/GVFSEnlistmentStatistics.cs

```diff
@@ -0,0 +1,189 @@
+﻿using GVFS.Common.Database;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace GVFS.Common
+{
+    public class GVFSEnlistmentStatistics
+    {
+        private List<string> gitPaths;
+        private List<string> placeholderFolderPaths;
+        private List<string> placeholderFilePaths;
+        private List<string> modifiedFolderPaths;
+        private List<string> modifiedFilePaths;
+        private int trackedFilesCount;
+        private int placeholderCount;
+        private int modifiedPathsCount;
+        private Dictionary<string, int> trackedFilesDirectoryTracking;
+        private Dictionary<string, int> hydratedFilesDirectoryTracking;
+
+        public GVFSEnlistmentStatistics(List<string> gitPaths, List<string> placeholderFolderPaths, List<string> placeholderFilePaths, List<string> modifiedFolderPaths, List<string> modifiedFilePaths)

```

Comment:
nit:  When you have lots of args, wrapping helps readability
        public GVFSEnlistmentStatistics(
                  List<string> gitPaths,
                  List<string> placeholderFolderPaths,
                  List<string> placeholderFilePaths,
                  List<string> modifiedFolderPaths, 
                  List<string> modifiedFilePaths)

Reply:
Made that change! Going to commit the changes once I get through all of the stuff